checkov/sca_package_2/runner.py

```diff
@@ -45,8 +46,16 @@ def prepare_and_scan(
             logging.info("The --bc-api-key flag needs to be set to run SCA package scanning")
             return None
 
+        if bc_integration.bc_source and bc_integration.bc_source.name in [BCSourceType.VSCODE, BCSourceType.JETBRAINS] \
```

Comment:
can you create a set in `checkov.common.bridgecrew.bc_source` with both values and reference it here, then we can directly extend there, if we support more someday and we can leverage it in other areas, if needed 🙂

Reply:
Fixed